src/HealthChecks.SendGrid/HealthChecks.SendGrid.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetStandardTargetVersion)</TargetFramework>
+    <PackageLicenseUrl>$(PackageLicenseUrl)</PackageLicenseUrl>
```

Comment:
This line makes no sense, the property overwrites itself. Same for Company, Authors and others.

Reply:
I copy pasted this line (section) from one other health check project :(

Reply:
I guessed. I was just waiting for a reason for someone to finally clean up these files.